manim/utils/config_utils.py

```diff
@@ -521,10 +508,72 @@ def finalized_configs_dict():
     return {section: dict(config[section]) for section in config.sections()}
 
 
-def _subcommands_exist(ignore=[]):
+def _subcommand_name(ignore=[]):
```

Comment:
!!! This should be an empty tuple !!!
Note that this default list is defined when the function is read. If this list is given out, it can be modified, and the default won't be an empty list anymore! So this should either be an empty tuple (so that there's no way this is going to be modified) or a factory (i.e., if None then default to [] inside the function).

```suggestion
def _subcommand_name(ignore=()):
```

Reply:
So, black flat out refuses to format the `(,)`, since it can't recognise what it means, for some reason?

I have modified it to `ignore=()`.